src/datepicker/__tests__/datepicker.test.js

```diff
@@ -151,4 +154,28 @@ describe('Datepicker', () => {
 
     expect(renderedInput.props().value).toEqual('2019/01/01 - 2019/01/04');
   });
+
+  test('calendar popover renders multiple months and hides redundant UI', () => {
```

Comment:
what ui is being hidden and what line here tests that? I would expect to see some tests related to hiding the arrows

Reply:
This test includes the multiple-month feature test and the hiding arrow test into one test, but based on this feedback I separated them into two distinct tests